src/tile/tile.scss

```diff
@@ -17,303 +17,332 @@
     color: t(iui-text-color);
   }
 
-  .iui-thumbnail {
-    width: 100%;
-    height: $iui-baseline * 14;
-    flex-shrink: 0;
-    overflow: hidden;
-    position: relative;
-    display: flex;
-    align-items: center;
-    justify-content: center;
+  .iui-tile-thumbnail-type-indicator {
```

Comment:
You could replace `.iui-tile-thumbnail-type-indicator` with `&-thumbnail-type-indicator`.

However I'm used to seeing all these mixins referenced within `classes.scss` rather than in the component's SCSS file.  Here's [an example from date-picker](https://github.com/iTwin/iTwinUI/blob/main/src/date-picker/classes.scss).

Reply:
I've made these changes.

Reply:
But forgot to push? :D

Reply:
I ran into a test issue last minute and held off on pushing.  Fixed it today so the changes should be available now.